README.md

```diff
@@ -72,94 +72,111 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 ### File explorer add-ons
 
-- Fixed an issue when creating thumbnails for SVG files created using Inkscape.
+- Quality of life improvements to Developer Files preview, including a progress bar while loading, performance improvements, dark mode, and logs. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Fixed possible WebView related vulnerabilities in the SVG and Markdown handlers.
+- Fixed some race conditions in Developer Files preview causing the loading bar to hang.
+- Added localization support to the Developer Files preview messages.
+- It's now possible to configure default color for Stl Thumbnails. Thanks [@pedrolamas](https://github.com/pedrolamas)!
+- Added an option to format JSON and XML files before rendering. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
-### Keyboard Manager
+### PowerRename
 
-- Adjusted wording on the editor when keys are orphaned.
+- Fixed an issue that was generating a silent crash when the context menu was triggered when not selecting any file or folder. (This was a hotfix for 0.61)
+- Improved performance when loading a big number of files.
+- Fixed a specific case in which PowerRename tried to rename a file to an empty string.
+- The UI now shows when a file can't be renamed due to its name being too long or containing invalid characters.
 
-### Mouse utility
+### PowerToys Run
 
-- Fixed a bug that caused the current Find My Mouse spotlight to hang when activated in the top left corner of the screen. (This was a hotfix for 0.60)
+- Added a fix to the VSCodeWorkspaces plugin to better support portable installations. Thanks [@bkmeneguello](https://github.com/bkmeneguello)!
+- The Folder plugin now expands `%HOMEPATH%` correctly.
+- Fixed a case where a previous result was being activated when searching for new results. Added a setting to better control input throttling. Thanks [@jefflord](https://github.com/jefflord)!
+- Added support for port numbers in the URI plugin. Thanks [@KohGeek](https://github.com/KohGeek)!
+- Fixed query errors when the search delay option was turned off.
+- New History plugin to search for old search results. Thanks [@jefflord](https://github.com/jefflord)!
+- Changed the default TimeDate activation keyword to `)` and added information in Settings so users know that some activation keywords may conflict with normal usage of some plugins when trying to do a global query. Thanks [@htcfreek](https://github.com/htcfreek)!
+- The Unit Converter plugin updated its UnitsNet dependency and now supports plural units. Thanks [@FWest98](https://github.com/FWest98)!
+- The Calculator plugin improved its input validation logic. Thanks [@htcfreek](https://github.com/htcfreek)!
 
-### PowerRename
+### Runner
 
-- The PowerRename window reacts to current dpi when created.
+- Clean up old install folders and logs at startup. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
```

Comment:
do we want to add something maybe like "this is a more aggressive cleanup method than we had prior"?

Reply:
Don't think that this has any relevance to the user. We can only mention that we also clean thing that were created before this change.

Reply:
100% does cause depending on how we upgraded, stuff wasn’t cleaned up and there have been a lot of feedback on this topic. This should solve all those issues now

Reply:
```suggestion
- Improved: Clean up old install folders and logs at startup. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
```
Minimal change, might be enough?